openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -4282,6 +4290,842 @@ def create_force(self, system, topology, **kwargs):
         system.addForce(gbsa_force)
 
 
+def validate_virtual_site_type_option(vsite_type):
+    """
+    Convert and validate the virtual site type specified in the VirtualSite element
+
+    Parameters
+    ----------
+    vsite_type : Any
+        The virtual site type to validate
+
+    Returns
+    -------
+    policy : str
```

Comment:
Just so it doesn't get overlooked before the merge -- Some copy-paste errors in docstrings here.